bgpd/bgp_routemap.c

```diff
@@ -3132,6 +3184,36 @@ DEFUN (no_match_mac_address,
 				      RMAP_EVENT_FILTER_DELETED);
 }
 
+DEFUN (match_evpn_route_type,
+       match_evpn_route_type_cmd,
+       "match evpn route-type {macip | multicast | prefix}",
```

Comment:
You probably want `<>` instead of `{}` here.

Reply:
make sense.